packages/client/src/v2-events/features/events/actions/correct/request/Review.tsx

```diff
@@ -59,14 +59,18 @@ export function Review() {
 
   async function handleEdit({
     pageId,
-    fieldId
+    fieldId,
+    skipConfirmation
```

Comment:
```suggestion
    confirmation = true
```

Would switching the api this way simplify the boolean logic or make it more complex?